Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -73,6 +73,55 @@
       margin-left: auto;
       margin-right: auto;
       margin-top: -40px;
+
+      .icons {
+        display: flex;
+        flex: 0 0 auto;

```

Comment:
is this necessary? i thought setting col/row would set thses

Reply:
using cols in a row creates two cols of equal width... we want to only take up the space that the inner content uses